internal/app/gateway/command/run.go

```diff
@@ -46,6 +50,13 @@ var gatewayCommand = &cobra.Command{
 	Run: run,
 }
 
+//PingData is to hold ping response
```

Comment:
Can't this just be in the ping service package itself? The gateway can construct it and pass it via Init.